src/api.js

```diff
@@ -0,0 +1,112 @@
+const express = require("express")
+const bodyParser = require('body-parser')
+const app = express();
```

Comment:
Use something like this to create APIs

https://www.npmjs.com/package/express-json-rpc-router

using some standard API communication format will mean we will not have to create an api-client ourselves in Python

anyway i prefer the clarity of jsonrpc over ambiguous REST API style

Reply:
Python API Client will also be part of your scope of work